server/routes/api.js

```diff
@@ -595,18 +595,18 @@ router.post('/messages', isLoggedIn, function(req, res, next) {
                           .as('temp_tab')
                     })
               })
-              .andWhere('message', 'like', message)
+              .andWhere('message', 'like', `%${message}%`)
```

Comment:
Thinking about this further, do these even need to be like's? the whole idea is we want to match exact messages anyway. Thoughts?

Reply:
Yknow... I think you might be onto something. If it's = instead of LIKE then it might do less dodgy shit with special chars in the string. And from my Googling it appears there's more weirdness with LIKE too: https://stackoverflow.com/questions/543580/equals-vs-like

Reply:
Pushed that change to my dupe heavy instance, looking good so far, see how we go

Reply:
I'd be curious if it makes a difference for #256 as well